### ISSUE TITLE
tmux version detection fails when version if prefixed

### ISSUE THREAD

**Issue by dragozir** [powerline/powerline#2167]
The tmux binding fails with tmux develop as it is currently of the form `next-3.3`. 

```
$ tmux -V
tmux next-3.3

Traceback (most recent call last):
  File "/home/jfh/.virtualenvs/default/bin/powerline-config", line 22, in <module>
    args.function(pl, args)
  File "/home/jfh/.virtualenvs/default/lib/python3.8/site-packages/powerline/commands/config.py", line 15, in __call__
    self.function(*args, **kwargs)
  File "/home/jfh/.virtualenvs/default/lib/python3.8/site-packages/powerline/bindings/config.py", line 184, in tmux_setup
    tmux_version = get_tmux_version(pl)
  File "/home/jfh/.virtualenvs/default/lib/python3.8/site-packages/powerline/bindings/tmux/__init__.py", line 84, in get_tmux_version
    return TmuxVersionInfo(int(major), int(minor), suffix)
ValueError: invalid literal for int() with base 10: 'next-3'
```
I'm currently building tmux as a conan package (not that it is particularly important) so I have control over both the software version as well as the source code so it isn't ablocking issue (for me anyways), but I noticed [this issue](https://github.com/powerline/powerline/issues/2150) from February describing a similar issue, and am wondering about possible approaches to fixing this? Should a more sophisticated approach be taken other than just stripping 'next' from the text, such as a regex to match all numbers separated by periods? Also perhaps an environment variable may help. Hopefully someone with more knowledge could provide context about any possible corner cases.

**Comment by dragozir** [comment]
Apologies, I see this was fixed in #1992. I guess a better question is where can I find a release for `>= 2.8`?

**Comment by PH111P** [comment]
See #2116.
